speech/snippets/transcribe_feature_in_recognizer.py

```diff
@@ -71,11 +90,6 @@ def transcribe_feature_in_recognizer(
 
 
 if __name__ == "__main__":
-    parser = argparse.ArgumentParser(
-        description=__doc__, formatter_class=argparse.RawDescriptionHelpFormatter
+    transcribe_feature_in_recognizer(
```

Comment:
As per the [style guide](https://googlecloudplatform.github.io/samples-style-guide/#no-cli) it's discouraged to provide a CLI interface on code samples.

Reply:
This issue solved by replacing the `region_tag` 🙌 Users will be able to see the complete example in the full version of Sample.